app/src/main/java/com/appunite/loudius/ui/components/LoudiusErrorDialog.kt

```diff
@@ -40,37 +35,17 @@ fun LoudiusErrorDialog(
 ) {
     var openDialog by remember { mutableStateOf(true) }
```

Comment:
question: why do we actually have this dialog? It's unused. Does anyone know? 🤔

Reply:
It's not used, but probable it should be when notifing :)